### PR TITLE
Detect and prompt if user navigates during an operation

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Detect and prompt if user navigates during an operation

Detect both navigation within and outside of the app

[stage-19]

## Motivation and Context
Network Management epic

## How Has This Been Tested?
Tested changes locally with various scenarios. Updated unit tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/scripts/common/directives/dtv-batch-operations.js [hunk 1]
```diff
@@ -52,6 +52,55 @@ angular.module('risevision.apps.directives')
...
+
+          var _bypass = false;
+          $scope.$on('$stateChangeStart', function (event, toState, toParams) {
+            if (_bypass) {
+              _bypass = false;
+              return;
+            }
+            if ($scope.listObject.operations.activeOperation) {
+              event.preventDefault();
+              var modalInstance = $modal.open({
```
Comment (ezequielc) [inline web/scripts/common/directives/dtv-batch-operations.js:64]: @alex-deaconu Nitpicking here, no need to update, but I would use `confirmModal` here. Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good 👍

File: web/scripts/common/directives/dtv-batch-operations.js (hunk 1 above)
Comment (alex-deaconu) [reply web/scripts/common/directives/dtv-batch-operations.js:64]: @ezequielc great point, however unfortunately this was the code I copied from the editor for the confirm modal and it had the old way of doing things. We can and should refactor later. Thanks.

**Pull request merged**